examples/demos/boot_from_SAN_and_server_profile_and_volume_migration/README.md

```diff
@@ -0,0 +1,17 @@
+# Demo
+## Server profile with boot from SAN and server profile & volume migration
+
+This demo is composed by 5 steps, each one associated with a playbook, that should be executed in the specified order: 
+
+- `step_1_create_server_profile_boot_from_SAN_and_assign_to_bay1.yml`: This playbook exemplifies how to create a server profile with boot from SAN (3PAR) and without using a template. On task #1, the server profile is created (if not already present) and assigned to server hardware in bay 1. You can check all parameters, including those related to the server hardware and those related to the storage as SAN connections and volume attachment. The volume must be ready to use. On task #2, the server hardware is powered on.
+
+- `step_2_based_on_server_profile_boot_from_SAN_add_another_volume.yml`: This playbook exemplifies how to add an extra volume attachment to an existing server profile. Task #1 was copied from step 1 and a new volume attachment was added identified by id 2. When this task is executed, the additional volume becomes available to the operating system.
```

Comment:
For the titles here I'd suggest linking the example, like:
[Step 2 - Add extra volume to existing server profile](https://github.com/HewlettPackard/oneview-ansible/pull/287/files#diff-0277cec457138aa03d8a9961b33111e4)

Reply:
Ok. Changed.